core/tests/protractor_utils/AdminPage.js

```diff
@@ -154,6 +154,54 @@ var AdminPage = function() {
     await waitFor.pageToFullyLoad();
   };
 
+  this.getFeaturesTab = async function() {
+    await this.get();
+    const featuresTab = element(by.css('.protractor-test-admin-features-tab'));
+    await waitFor.elementToBeClickable(
+      featuresTab, 'Features tab is not clickable');
+    await featuresTab.click();
```

Comment:
Can we use action.click? See example below:

https://github.com/oppia/oppia/blob/b9927cc04321f6def1047d4a44336230f2b77ada/core/tests/protractor_utils/AdminPage.js#L313

Reply:
Yes it's more feasible, used `action.click` instead.